spec/lib/dqt/params/trn_request_spec.rb

```diff
@@ -164,8 +164,8 @@ module Params
           end
         end
 
-        context "when gender is gender_not_provided" do
-          let(:trainee) { create(:trainee, :completed, gender: "prefer_not_to_say") }
+        context "when sex is gender_not_provided" do
```

Comment:
super minor comment, looks like this context is wrong, should it read:

```
context "when sex is prefer_not_to_say" do
```

it looks like the test context was wrong originally too!

Reply:
Looking into this I think the TrnRequest class was mapping for a non-existent enum for gender/sex (`gender_not_available` vs `gender_not_provided` as we have it defined in `Trainee`. 

I've changed this and added an extra spec

<img width="331" alt="Screenshot 2022-09-22 at 10 40 01" src="https://user-images.githubusercontent.com/6339025/191713911-250c4673-b3e6-43b9-92cd-27c446b8cbfd.png">

Reply:
Good spot!